internal/idp/keycloak/authz_groups.go

```
@@ -0,0 +1,169 @@
+	if err := json.NewDecoder(response.Body).Decode(&scopes); err != nil {
+		return "", fmt.Errorf("failed to decode scopes: %w", err)
+	}
+
+	if len(scopes) == 0 {
+		return "", fmt.Errorf("scope %s not found", scopeName)
+	}
+
+	return scopes[0].ID, nil
+}
+
+func (c *Client) getPermissionByName(ctx context.Context, clientUUID, permissionName string) (string, error) {
```

golangci-lint (2.12.2)

[error] func (*Client).getPermissionByName is unused

(unused)